app/controllers/admin/domains_controller.rb

```diff
@@ -63,8 +63,11 @@ def update
     # rubocop:enable Metrics/MethodLength
 
     def versions
+      per_page = 10
```

Comment:
Looks like some magic number to me. Consider using a constant like DEFAULT_RECORDS_PER_PAGE and defining it in the controller itself.

Reply:
Thanks! Just saw the same apporach in other controllers. Actually I wanted to add paginates_per 10 to the model itself, but did not want to make it general rule. I will do like you suggested - add constant in the controller!